content/docs/lists-and-keys.md

```diff
@@ -202,13 +206,14 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
+[**Coba di CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
 
-A good rule of thumb is that elements inside the `map()` call need keys.
+Sebuah aturan yang mudah diingat adalah elemen di dalam pemanggilan `map()` akan membutuhkan *key*.
 
-### Keys Must Only Be Unique Among Siblings {#keys-must-only-be-unique-among-siblings}
+### *Key* Harus Bersifat Unik Diantara Saudaranya {#keys-must-only-be-unique-among-siblings}
```

Comment:
Loanwords in header shouldn't be italicized 😄

Reply:
Good to know. Might need to double check all ###headings with loanwords in it.